en/admin/config.rst

```diff
@@ -2404,6 +2408,23 @@ The following are other parameters. The type and value range for each parameter
 
  It specifies whether option information in the **WITH** clause is displayed when index syntax information is displayed, such as in the SHOW CREATE TABLE statement. Default is NO. However, the unloaddb tool is not affected by this setting.
 
+**enable_memory_monitoring**
+
+ **enable_memory_monitoring** specifies whether to monitor the server's heap memory usage. Setting the value to YES, the server's memory monitoring feature is activated, which continuously tracks and manages the server's heap memory usage. Heap memory usage is tracked based on the file and line where dynamic memory allocation occurs in the CUBRID source code. If multiple memory allocations occur at the same location, the memory usage is accumulated. When the tracked memory is deallocated, the amount of deallocated memory is subtracted from the accumulated total, continuously tracking the real-time heap memory usage. The monitored heap memory usage can be checked using :ref:`memmon` utility. The default value is NO.
+
+.. note::
+
+    *   It is only supported on Linux.
+    *   During the automated process of tracking memory usage at the code level using macros, conflicts with glibc occurred. To prevent this, memory usage within glibc(STL containers) is not tracked. And for the same reason, memory usage occurring in header files is not tracked.
```

Comment:
```suggestion
    *   During the automated process of tracking memory usage at the code level using macros, conflicts with glibc occurred. To prevent this, memory usage within glibc(STL containers) is not tracked, and memory usage occurring in header files is not tracked for the same reason.
```